src/components/atoms/menuItem/MenuItem.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "../../../foundations/sass/all";
+
+.menu-item {
+  // background-color: $white-floral;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  gap: 0.75rem;
```

Comment:
Can you retrieve the value of "gap" from the "spacing" variable?

Reply:
sure thing!